plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/AppEngineProjectWizard.java

```diff
@@ -40,7 +40,7 @@
 
 public abstract class AppEngineProjectWizard extends Wizard implements INewWizard {
 
-  private AppEngineWizardPage page = null;
+  protected AppEngineWizardPage page = null;
```

Comment:
See Effective Java 2E, p. 69. This is borderline and may affect thread safety. Making this private with a final setter and getter would make me a lot less nervous.